omnigibson/scenes/interactive_traversable_scene.py

```diff
@@ -51,6 +52,7 @@ def __init__(
             not_load_object_categories (None or list): if specified, do not load these object categories into the scene
             load_room_types (None or list): only load objects in these room types into the scene
             load_room_instances (None or list): if specified, only load objects in these room instances into the scene
+            load_task_relevant_only (bool): Whether only task relevant objects (and building structure) should be loaded
```

Comment:
I don't know if I agree with this paradigm. A scene is task-agnostic, so how would it know what is task-relevant if it's being loaded separately from a task instance?

I definitely agree with the high-level use case, but i think this flag should exist elsewhere, eg..: at the env class level, or somewhere which owns both a task and scene instance